Shared/View/BaseItemListView.swift

```diff
@@ -84,21 +84,29 @@ class BaseItemListView: UIViewController {
     }
 
     internal func styleNavigationBar() {
```

Comment:
Are there any other places this method could go. Should it go in a common `UIViewController+` extension, and called as part of `viewDidLoad`?

Reply:
Yes - I would like to refactor all of the navigation styling in a later PR.